Tracker/tracker.lua

```diff
@@ -1328,6 +1328,7 @@ function tracker:loadKeys( keySet )
       { 'CTRL + Shift + +/-', 'Add CC (adv mode)' },
       { 'CTRL + Shift + A/P', 'Per channel CC/PC' },
       { 'CTRL + Shift + Click row indicator', 'Change highlighting (RMB resets)' },
+      { 'CTRL + Alt + Click row indicator', 'Change fixed indicator position' },

```

Comment:
Might be worth a mention in the README also? It sounds a bit obscure with only this as a explainer.

Reply:
Yes, for sure. Please feel free to add as much info as you think is needed.

There are a few small other things that would be good to do before release (I should probably make a PR template):
- Add an entry to the changelog. It can be multi-line, but make sure it adheres to the format to prevent issues with reapack.
- Bump the version number (note that it is in two places, in the header and in the code `tracker.name` (I often forget to update the latter x_x)).
- Add your name to the author list under `@author` (if you want) :)

I think that's it.

Reply:
Cool, will do.

Reply:
Rebased, bumped version and added short section to README.txt with screen-grab. I can't claim co-authorship but appreciate the offer!

Reply:
👍